perfkitbenchmarker/benchmarks/object_storage_service_benchmark.py

```diff
@@ -572,8 +604,10 @@ def Run(self, vm, metadata):
 
     # Now tests the storage provider via APIs
     test_script_path = '%s/run/%s' % (scratch_dir, API_TEST_SCRIPT)
+    hostname = AWS_REGION_TO_ENDPOINT_TABLE[FLAGS.object_storage_aws_region]
     ApiBasedBenchmarks(results, metadata, vm, 'S3', test_script_path,
-                       self.bucket_name)
+                       self.bucket_name,
+                       host_to_connect=hostname + '.amazonaws.com')
```

Comment:
make the .amazonaws.com part as a constant.

In general, as we discussed in previous CRs, consider define all literal strings as constants. (to avoid typos, and easy for lookup)